docs/src/Master_Developer.txt

```diff
@@ -1,9 +1,11 @@
 :lversion: {sys: cat ../VERSION}
+:localdate: {sys: egrep "^ *--" debian/changelog | head -n1 | sed -e 's/^.*  //' | cut -f2,3,4 -d' '}
```

Comment:
I don't think you can rely on the build happening in the root of the repo, can you?
How about something like this:
```
date --date="@$(dpkg-parsechangelog --file $(git rev-parse --show-toplevel)/debian/changelog -S timestamp)" '+%d %b %Y'
```

Reply:
I like it. And thank you for your critical thinking and improvements. If I guess right then you have this already implemented locally and do not need anything patched from my side!?

Reply:
> I don't think you can rely on the build happening in the root of the repo, can you?
Debian-wise everything starts in the root of the source tree. But I admittedly completely forgot that the documentation is likely built in a subdirectory. Not good. 

> How about something like this:
> 
> ```
> date --date="@$(dpkg-parsechangelog --file $(git rev-parse --show-toplevel)/debian/changelog -S timestamp)" '+%d %b %Y'
> ```

```
$ date '+%d %b %Y'
29 Okt 2021
$ LANG=C date '+%d %b %Y'
29 Oct 2021
```
Hence I propose
```
LANG=C date --date="@$(dpkg-parsechangelog --file $(git rev-parse --show-toplevel)/debian/changelog -S timestamp)" '+%d %b %Y'
```
to be reproducible across continents. To avoid an English date in a French manual  we could in some deeper theory also adjust the LANG for each language. But I want to read the date also for the Chinese manual to ensure it is the latest version, so maybe it is just fine to have LANG=C everywhere.

Reply:
What if the build happen from a tarball of the source, and not the git repository?

Reply:
Oops, good catch!  I should have read your comment before merging just now...
Our orig.tgz currently includes the git repo, but that's probably wrong.  It's certainly bulky..,
I'll see about a fix.

Reply:
BTW the commits I just merged are 479df775c1c86abf6f52e1de3423f21847514e84 and 82eeb08bca72d02dd2ccf3e47f2a7a11644921d2.

Reply:
> Our orig.tgz currently includes the git repo, but that's probably wrong. It's certainly bulky..,

Oops, no it doesn't, i just messed up locally.  The tarballs that our buildbot makes (which go with the dscs we publish to our users) do *not* include the .git directory.

So the reliance on git tools in my changes means the date is messed up in our PDFs now.  I'll fix it.